app/Tests/CourseControllerTests.hs

```diff
@@ -0,0 +1,55 @@
+{-|
+Description: Course Controller module tests.
+
+Module that contains the tests for the functions in the Course Controller module.
+
+-}
+
+module Tests.CourseControllerTests
+( courseContTestSuite ) where
+
+import Config (runDb)
+import Controllers.Course (index)
+import qualified Data.ByteString.Lazy.Char8 as BL
+import Data.Text as T
```

Comment:
use `qualified` here as well; I think that should stop the collision of `map`, so you can use just `map` and not `Prelude.map` down below